doc/source/plans_intro.rst

```diff
@@ -73,9 +73,16 @@ Or, similarly, we can plot the data during the scan using ``LivePlot``.
 
 .. ipython:: python
 
+    import matplotlib.pyplot as plt; plt.ion()
     from bluesky.callbacks import LivePlot
     RE(scan([det], motor, 1, 5, 5), LivePlot('det', 'motor'))
 
+.. note::
+
+    You'll want to make sure your plotting system is in interactive mode to see
+    the plots update in real time. This is what the first line effectively
+    accomplishes.
```

Comment:
Should we add a note that if you are using a jupyter notebook then you should use ```%matplotlib ...``` ?

Reply:
We have it [here](https://nsls-ii.github.io/bluesky/callbacks.html#aside-making-plots-update-live). I think I'd prefer to _link_ to that note so that we only have to update it in one place if our advice changes in the future (which I expect it will with Jupyter _Lab_).